src/directives/filereader.js

```diff
@@ -15,6 +15,8 @@ goog.require('ngeo');
  *      <input type="file" ngeo-filereader="ctrl.fileContent"
  *        ngeo-filereader-supported="ctrl.supported"/>
  *
+ * @htmlAttribute {string} ngeo-filereader The content of the file read
+ * @htmlAttribute {boolean} ngeo-filereader-supported Whether the FileReader API is supported
```

Comment:
This attribute is explicitly declared as "optional" (`=?ngeoFilereaderSupported`, with a "?"). Perhaps we can say: `{boolean} ngeo-filereader-supported  Optional attribute. Whether the FileReader API is supported`.